pftools/python/parflow/tools/builders.py

```diff
@@ -311,6 +311,10 @@ def __init__(self, run=None):
         self.column_index = {}
         self.props_in_row_header = True
         self.table_comments = []
+        self.empty_value_tokens = {
+            "-",
+            ""
+        }
```

Comment:
Make it so that the empty string also counts as an empty value. This supports the use case of leaving cells blank in a csv